tests/precommit_test.py

```diff
@@ -131,22 +134,35 @@ def test_get_added_and_existing_findings_in_changes_for_changes(self):
         self.assert_findings_ids(self.precommit_client.findings_in_changed_code, [])
         self.assert_findings_ids(self.precommit_client.existing_findings, [4, 5, 6, 7])
 
+    @responses.activate
+    def test_adding_path_prefix(self):
+        path_prefix = 'prefix/'
+        self.precommit_client = self._get_precommit_client(self._get_changed_file(),
+                                                           self._get_no_deleted_files(), path_prefix=path_prefix)
+        self.mock_precommit_findings_churn(added_findings=[1], path_prefix=path_prefix)
+        self.precommit_client.run()
+
+        self.assert_findings_paths(self.precommit_client.added_findings, [path_prefix + ANALYZED_FILE_NAME])
```

Comment:
I'm not sure what this asserts 😅
I would expect a test that checks whether the extended paths are sent to the server and a test that ensures the stripped paths are returned. Does this test one (or both) of these aspects?

Reply:
Nothing really, good catch ^^ Fixed it and added a test for stripping the prefix again :)